src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormattingPass.cs

```diff
@@ -71,8 +71,6 @@ public async override Task<FormattingResult> ExecuteAsync(FormattingContext cont
             cancellationToken.ThrowIfCancellationRequested();
 
             // We make an optimistic attempt at fixing corner cases.
-            var cleanupChanges = CleanupDocument(changedContext);
```

Comment:
Are we able to remove the "CleanupDocument" code?

Reply:
`CleanupDocument` is still used in `CSharpOnTypeFormattingPass`, and the RazorFormattingPass doesn't run on OnType yet, so `CleanupDocument` stays for now (unless I misunderstood @davidwengier).

Reply:
100% correct